src/providers/DevicesProvider/VideoInputProvider.tsx

```diff
@@ -17,7 +17,7 @@ import { useMeetingManager } from '../MeetingProvider';
 
 const Context = createContext<VideoInputContextType | null>(null);
 
-const VideoInputProvider: React.FC<React.PropsWithChildren<unknown>> = ({
+export const VideoInputProvider: React.FC<React.PropsWithChildren<unknown>> = ({
```

Comment:
Any reason to change these exports or just a better way?

Reply:
We migrated from `Props` to `ArgTypes` for stories. However `ArgTypes` can only read the props of a component when this component is `export`. `default export` does not work. I linked this storybook issue in the commit message.
>Check this storybook issue: https://github.com/storybookjs/storybook/issues/13304#issuecomment-745360896.
But this is not a breaking change because we exported them already.
Btw I did the same change to icons.

Reply:
So basically if we don't export the component, the props table will be empty.

Reply:
Yup not breaking change for sure and I just want to understand if there was a reason, thanks for explanation!